luaui/Widgets/unit_area_reclaim_enemy.lua

```diff
@@ -0,0 +1,125 @@
+
+local widget = widget ---@type Widget
+function widget:GetInfo()
+	return {
+		name			= "Area Reclaim Enemy",
+		desc			= "Hold down Space an give area reclaim order on the ground or enemy to target enemies only during reclaim.",
+		author			= "NemoTheHero",
+		date			= "July 26, 2025",
+		license			= "GNU GPL, v2 or later",
+		layer			= 0,
+		enabled			= true
+	}
+end
+
+local allyTeam = Spring.GetMyAllyTeamID()
+
+-- Speedups
+
+local spGiveOrderToUnitArray = Spring.GiveOrderToUnitArray
+local spGetSelectedUnits = Spring.GetSelectedUnits
+local spGetUnitsInCylinder = Spring.GetUnitsInCylinder
+local spWorldToScreenCoords = Spring.WorldToScreenCoords
+local spTraceScreenRay = Spring.TraceScreenRay
+local spGetUnitDefID = Spring.GetUnitDefID
+local spGetUnitAllyTeam = Spring.GetUnitAllyTeam
+local spGetUnitCmdDescs= Spring.GetUnitCmdDescs
+local spGetUnitPosition= Spring.GetUnitPosition
+
+local reclaimEnemy = Game.reclaimAllowEnemies
+
+local CMD_RECLAIM = CMD.RECLAIM
+
+
+
+function maybeRemoveSelf()
+    if Spring.GetSpectatingState() and (Spring.GetGameFrame() > 0) then
+        widgetHandler:RemoveWidget()
+    end
+end
+
+function widget:GameStart()
+    maybeRemoveSelf()
+end
+
+function widget:PlayerChanged(playerID)
+    maybeRemoveSelf()
+end
+
+function widget:Initialize()
+    if Spring.IsReplay() then
+        maybeRemoveSelf()
+    end
+end
+
+function widget:CommandNotify(id, params, options)
+	-- RECLAIM with area affect
+	if id ~= CMD_RECLAIM or #params ~= 4 then
+		return
+	end
+	-- holding space key
+	if options.meta then
```

Comment:
Could do an early `if not meta then return` exit and save an indentation level in the rest of the function